src/components/heroSection/Hero.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import React from "react";
+import HeroImage from "../../../public/assets/images/heroImage.svg";
+import Search from "../../../public/assets/Search.svg";
+
+const Hero = () => {
+  return (
+    <div className=" w-full h-full bg-[#eaeaea] ">
+      <div className="w-full h-screen bg-[url('https://s3-alpha-sig.figma.com/img/be3d/4572/95d73f726199a6e31e24f7b63daa1a82?Expires=1676851200&Signature=pjRJnadNsA7JGPWF0d8EnX68CS5-ESV9OAd0EdcL1~htONJGvZ1nBHWGrlRbNAQ80z9gC08jPTGbyEMKXrigcAt~znkhh0m~vt~1cZ5mQVT97oSdzcqMtXaKT2uozddzXXRc2nW-X~viy5VSWcJmulFrEGN69RIufX0dcCxJAKr7tZjxFM7OMWeHXDTmNeu~Rys8B5Fr3nAoc8AobfMneyMgAA22mNTkU6AHTDFIWl0TAD7f494RUeFaX3meJ8AQcPx6yZkKn-y3V40C9lxryK18knov8ATO30rRjBk3ya8xmOjbZSBGBfY36Rb346HADSh2cM7uuDxvqF504Bvndw__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4')] bg-cover bg-center">
+        <div className="max-w-[1210px] mx-auto p-10 flex flex-col lg:flex-row justify-between items-center">
+          <div className="w-full max-w-[650px]">
+            <p className=" text-4xl md:text-6xl font-[750] md:leading-[70px]">
+              Where great <span className="text-[#7AB8F1]">Jobs</span> and great{" "}
+              <span className="text-[#A38DFD]">Engineers</span> come to find one
+              another !
+            </p>
+            <p className="py-6 text-base">
+              So many jobs available, all you have to do is keep up with our
+              posts. Check below for recent job openings.{" "}
+            </p>
+            <div className="bg-white p-2  flex justify-between items-center rounded-lg">
+              <div className="flex items-center">
+                <Image src={Search} alt="/" />
```

Comment:
Please change this from using an image to use the icon component in the atoms folder. This project is currently using React Icons for all it's Icons and so please use a Search Icon from that library. There are example usages of the Icon component in the footer to help you use it. Also delete the search svg once you've finished the immigration please.

Also, could you add the alt text, 'Search Icon' when you make that change please.

Reply:
Just a quick reminder to change the this search image to an icon please? Thank you so much!